docs/How-We-Work/team-charter/sprint-schedule.md

```diff
@@ -1,58 +1,60 @@
 # Sprint Schedule
 
-_This is just a quick summary of each day's meetings and focus. See [Team Meetings](https://github.com/HHS/TANF-app/blob/SelenaJV-patch-1/docs/team-charter/team-meetings.md) and [Our Workflow](https://github.com/HHS/TANF-app/blob/SelenaJV-patch-1/docs/team-charter/our-workflow.md) for more details._
+_This is just a quick summary of each day's meetings and focus. See [Team Meetings](../team-meetings.md) and [Our Workflow](../our-workflow.md) for more details._
  
 ### Wednesday Sprint Day 1: Planning Day!
-- 10:00-11:00am EST: Backlog Refinement 
-- NO STANDUP!
-- 3:30-5:00pm EST: Sprint Planning  
+- 10:30-11:30am EST: Backlog Refinement 
+- Async Standup
+- 2:00-2:30pm EST: Dev Sync
+- 3:30-4:30pm EST: Sprint Planning  
+- Sprint goals are discussed in refinement and reviewed during sprint planning. The team uses these goals to prioritize work for the sprint.
 
 ### Thursday Sprint Day 2: Work Day!   
 - 10:00-11:30am EST: OFA + 18F Coordination
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
 - Request feedback from teammates on rolling basis
 
 ### Friday Sprint Day 3: Work Day!   
-- 10:30-10:45am EST: Standup
+- 12:00-1:00pm EST: IPT
+- Async Standup
+- 2:00-2:30pm EST: Tech Lead Sync
 - Request feedback from teammates on rolling basis 
 
 ### Monday Sprint Day 6: Work Day!  
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
+- 12:15-12:45pm EST: UX Sync
 - Request feedback from teammates on rolling basis 
 
 ### Tuesday Sprint Day 7: Work Day!  
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
 - Request feedback from teammates on rolling basis 
 
-### Wednesday Sprint Day 8: Raft Review Day!
-- 10:00-11:00am EST: Backlog Refinement  (Security focused)
-- NO STANDUP!
-- Request Raft review for issues that will likely be including in current sprint's PR    
-
+### Wednesday Sprint Day 8: Work Day!
+- 10:00-10:30am EST: Security Sync
+- 10:30-11:30am EST: Backlog Refinement
+- 12:00-12:15pm EST: Standup
+- 2:00-2:30pm EST: Dev Sync
+- Request feedback from teammates on rolling basis
+    
 ### Thursday Sprint Day 9: Work Day!  
 - 10:00-11:30am EST: OFA + 18F Coordination
-- 10:30-10:45am EST: Standup
+- 12:00-12:15pm EST: Standup
 - Request feedback from teammates on rolling basis 
 
-### Friday Sprint Day 10: Submit Day!
-- 10:30-10:45am EST: Standup
-- Last day to request government review for work to count in this sprint 
+### Friday Sprint Day 10: Work Day!
```

Comment:
@kniz-raft every friday, dev team has optional drop-in hours from 10:30a-12:00p (ET).

Reply:
Updated - thanks!